docker-nix/.gitignore

```diff
@@ -0,0 +1 @@
+genfiles
```

Comment:
I'd rather override `/docker/e2e` than create a new root directory. I'd also be okay with `/docker/nix`.

Reply:
Basically, either move this to `/docker/nix` or just change the `e2e` docker files to use nix.